.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,66 @@
+# Sample workflow for building and deploying a VitePress site to GitHub Pages
+#
+name: Deploy Backend Documentation to Pages
+
+on:
+  # Runs on pushes targeting the `main` branch. Change this to `master` if you're
+  # using the `master` branch as the default branch.
+  push:
+    branches: [master]
```

Comment:
This is the one line that we should discuss in the team - when to build docs.

Reply:
That's also the current setting for the jekyll docs so nothing is really changing. This could also be discussed seperately from this PR.